spec/index.bs

```diff
@@ -2169,6 +2173,34 @@ interface GPUTextureUsage {
 };
 </script>
 
+<div algorithm>
+    <dfn abstract-op>maximum mipLevel count</dfn>(dimension, size)
+    **Arguments:**
+        - {{GPUTextureDescriptor/dimension}} |dimension|
+        - {{GPUTextureDescriptor/size}} |size|
+
+    1. Calculate the values of |w|, |h|, and |d|:
+        - If |dimension| is:
+            <dl class="switch">
+                : {{GPUTextureDimension/"1d"}}
+                :: Let |w| = |size|.[=Extent3D/width=].
+                :: Let |h| = 1.
+                :: Let |d| = 1.
+
+                : {{GPUTextureDimension/"2d"}}
+                :: Let |w| = |size|.[=Extent3D/width=].
+                :: Let |h| = |size|.[=Extent3D/height=].
+                :: Let |d| = 1.
+
+                : {{GPUTextureDimension/"3d"}}
+                :: Let |w| = |size|.[=Extent3D/width=].
+                :: Let |h| = |size|.[=Extent3D/height=].
+                :: Let |d| = |size|.[=Extent3D/depthOrArrayLayer=].
+            </dl>
+    1. Let |m| = the maximum value of |w|, |h|, and |d|.
+    1. Return one plus the greatest integral value of |x| for which 2<sup>|x|</sup> < |m|.
```

Comment:
Perhaps it would be more helpful to simply describe this with an actual formula. `floor(log2(m)) + 1` I believe

Reply:
Both look fine to me. The small appeal for the way it's written right now is that it's integer arithmetic.

Reply:
Done. I followed Kai's suggestion. And the formula is correct.